web/app/mu-plugins/pantheon-loader.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Plugin Name: Pantheon MU Plugin Loader
+ * Description: Loads the MU plugins required to run the site
+ * Author: Pantheon
+ * Author URI: https://pantheon.io
+ * Version: 1.0
+ * License: MIT
+ *
+ * IMPORTANT NOTE:
+ * Do not modify this file. This file is maintained by Pantheon.
+ *
+ * If you wish to add mu-plugins, please create your own PHP loader
+ * file inside the mu-plugins directory. This file may change in 
+ * future releases and modifications would cause conflicts when attempting
+ * to apply upstream updates.
+ */
+
+if ( ( defined( 'WP_INSTALLING' ) && WP_INSTALLING ) ) {
+	return;
+}
+
+// Add your mu-plugins here.
+$mu_plugins = [
+	'bedrock-disallow-indexing/bedrock-disallow-indexing.php',
+	'pantheon-mu-plugin/pantheon.php',
+];
+
+foreach ( $mu_plugins as $file ) {
+	$path = dirname(__FILE__) . 'loader.php/' . $file;
+
+	if ( file_exists( $path ) ) {
+		include_once $path;
+	} else {
+		error_log( sprintf( 'Tried to load %s but the file was not found.', $file ) );
+	}
+}
+unset( $file );
+
+add_action(
+	'pre_current_active_plugins', function () use ( $mu_plugins ) {
```

Comment:
Where is this from?

Reply:
As far as I can tell, the origin of this code and the filter below (`network_admin_plugin_action_links`) is here: https://github.com/pantheon-systems/wp13n/commit/7bd722d2f8282db0258cbc2ae435c5a48274d2f7

And was later moved to [wordpress-bedrock-recommended](https://github.com/pantheon-systems/wordpress-bedrock-recommended/commit/66a621cba4ee20cecd51bc71f982590124d4e9b2#diff-d9815a88be153f1989fd34f7cb7841cbc844d1ac1e67769ac24318555922f991) and finally to this prototype. These filters are used to render the mu-plugins in the site plugin list at `wp/wp-admin/plugins.php?plugin_status=mustuse`, and I don't see any issue with including this.

Reply:
This loader file is a variation of the one we used at Human Made, which looks something like this (note, this is an older version, but it's probably still pretty close): https://github.com/humanmade/hm-base/blob/master/content/mu-plugins/loader.php